sections/06_discussion.md

```diff
@@ -6,22 +6,10 @@ can be further pruned.*
 
 ### Evaluation
 
-*What are the challenges in evaluating deep learning models that are specific to
-this domain?  This can include a discussion of ROC versus precision-recall
-curves for the imbalanced classes often encountered in biomedical datasets.
-It could also mention alternative metrics that are used in specific sub-areas
-such as enrichment factors in virtual screening.  A lack of true gold standard
-data for some problems complicates both training and evaluation.  Confidence-
-weighted labels are valuable when available.*
-
-*Is progress in some biomedical areas slowed when new predictions (e.g. from
-generative models) cannot be assessed by any human expert and require
-experimental testing?  For example, contrast a painting or song generated by
-a GAN versus a novel chemical compound.  Related is the idea that on some tasks
-(e.g. the recent wave of deep learning versus MD image classification papers)
-it is easy to tell when deep learning has produced a breakthrough because
-human-level performance is an impressive baseline.  In many tasks we reviewed,
-human-level performance is irrelevant.*
+The human genome has 3E9 base pairs (bp) and only a small fraction of them are implicated in specific biochemical activities. As a result, binary classification of genomic regions based on their biochemical activity results in small positive class and a large negative class with important consequences for model evaluation.  For example, based on processed ChIP-seq data, the number of reproducible transcription factor (TF) binding sites genome-wide is typically in the 1,000-50,000 range. ChIP-seq data allows TF binding site detection with ~200 bp resolution and thus less than 1% of the genome can be confidently labeled as bound for most transcription factors. Therefore, it is important evaluate the genome-wide recall and false discovery rates (FDR) of computational models of TF binding. TF binding site predictions may be used for targeted validation experiments and as features for models of downstream processes such as gene expression. Targeted experimental validation usually necessitates an FDR in the 5%-25% range. For downstream modeling, a low FDR for TF binding predictions is not as critical if the downstream models meet their respective recall and and FDR requirements; an FDR of 50% in this context may suffice.
```

Comment:
We should reframe this to answer the question above: "What are the challenges in evaluating deep learning models that are specific to this domain?" That prompt arose from this discussion in #43 and #109 for context.  The TF binding problem is a very good motivating example, as you describe here, but we should present it as an example of the broader topic.  I suggest leading with a discussion of class imbalance and possibly other pitfalls that are sometimes ignored when evaluating predictive models.  Then you can transition to some of what you have here illustrating how it is a problem for TF binding.  That can include a few of these details about just how imbalanced this prediction task is.

It will also strengthen our case to refer to previous work that supports our claim that ROC is inappropriate for imbalanced datasets.  http://www.nature.com/nmeth/journal/v13/n8/full/nmeth.3945.html and http://dl.acm.org/citation.cfm?doid=1143844.1143874 are especially relevant.  http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0118432 came up before, but it isn't critical if we use those other two.